Design/Instrument-2.0/ScanningInstruments_Design.md

```diff
@@ -0,0 +1,102 @@
+## Scanning Instrument Support for the ILL
+
+### Background
+
+#### Instrument 2.0
+
+The full discussion on Instrument 2.0 is beyond the scope of this document, more details are contained in this [requirements document](requirements-v2.md). A short summary on [scanning instrument requirements is given](requirements-v2.md#moving-instruments-requirements) within this document.
+
+For the purposes of the discussion here scanning instruments (or moving instruments) describe instruments that have detectors that can be in different positions during a single run. This forms a step scan. This is used to cover data collection where there are gaps between detectors. The aim of the work is to be able to deal with such detectors that move during a run, and be able to accumulate the data into a single workspace with the detectors at different positions throughout the run. This does not include two other use cases from the Instrument 2.0 requirements document; continuous scans and triple axis spectrometers.
```

Comment:
Yes. Definition is right. For `Instrument 2.0` we would probably also want to consider scenarios where other components (non-detectors) could be scanned. I discussed this with @SimonHeybrock and our conclusion is that this should not be the focus of the immediate work and would very much be tied up in the full `Instrument 2.0` rollout itself.

Also. @MichaelWedel and I happen to think that `Instrument-ng` `ng` = next generation, might be a better name. Especially since I started versioning stages of the prototype.